### ISSUE TITLE
Gitea Support

### ISSUE THREAD

**Issue by jolheiser** [Clivern/Rabbit#63]
I've seen this posted on reddit a few times, and it looks pretty nice. 
It would be awesome to have integration with [Gitea](https://gitea.io/)!

**Comment by Clivern** [comment]
Yes, will add gitea support checked the webhooks and it seems pretty similar to github

Ref -> https://docs.gitea.io/en-us/webhooks/

**Comment by c-nv-s** [comment]
I am assuming this project is only in maintenance mode?
